src/Microsoft.Data.Sqlite/SqliteConnection.cs

```diff
@@ -8,6 +8,9 @@
 
 namespace Microsoft.Data.Sqlite
 {
+    /// <summary>
+    /// Represents a connection with a SQLite database
```

Comment:
Recommend ensuring all the doc comments are full sentences (I think just missing periods in some places).